src/connector/src/schema/schema_registry/client.rs

```diff
@@ -160,6 +177,12 @@ impl Client {
         self.get_subject(subject).await.map(|s| s.schema)
     }
 
+    // used for connection validate, just check if request is ok
+    pub async fn get_config(&self) -> SrResult<()> {
+        let _: GetConfigResp = self.concurrent_req(Method::GET, &["config"]).await?;
+        Ok(())
+    }
```

Comment:
Given this function does not really return a `Config` and is documented `used for connection validate`, it shall be named `validate_connection` with `struct GetConfigResp` defined inside, unless we plan to actually use the config.